io.c

```diff
@@ -2630,22 +2650,23 @@ rb_io_descriptor(VALUE io)
 
 /*
  *  call-seq:
- *     ios.pid    -> integer
+ *    pid -> integer
```

Comment:
`integer` or `nil`

Somewhere in the description, we should describe that this returns `nil` for instances not created by `IO.popen`.

Reply:
Done.  I saw the nil in the previous doc, but could not figure out when it occurs.

Reply:
I think the call-seq should still be updated to say `integer or nil`.